posthog/api/decide.py

```diff
@@ -141,7 +141,10 @@ def get_decide(request: HttpRequest):
                     ),
                 )
 
-            property_overrides = get_geoip_properties(get_ip_address(request))
+            property_overrides = {}
+            if (data.get("geoip_disable") or False) is False:
```

Comment:
```suggestion
            if not data.get("geoip_disable"):
```

^^ Also test that this parameter is parsed correctly, no string to boolean issues (say I send it with 'true' instead of 'True'